### DISCUSSION TITLE
Problem in combination with a Slug field

### DISCUSSION THREAD

**Discussion by milenishte** [spatie/nova-translatable#96] [Q&A]
When a field is translatable, it can't trigger the automatic slug generation. I tried giving the Slug field  ->preview('title'), ->preview('title.en') and also passing the field as an an object, that is previously defined like
$title = Text::make('Title') ->translatable();
... ->preview($title);

**Comment by samigitz** [comment]
if anyone having this issue, try below

Slug::make('Slug') ->from('translations_title_en')

This is working
